Final Project/scraperTest.py

```diff
@@ -1,40 +1,72 @@
 """
 unit tests for scraper
 """
-
 #import response
 import unittest
-
+import requests
+import csv
+import os.path
+import re
+import time
+from bs4 import BeautifulSoup
+from selenium import webdriver
 import scraper as myScraper
 
+dataFilename='trainingCards.txt'
+url = myScraper.randomLinkGenerator()
```

Comment:
We should also use a dummy URL in case randomLinkGenerator fails

Reply:
Dummy URL used